spec/features/check_answers_page_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "Check answers page" do
+  let(:estimate_id) { SecureRandom.uuid }
+
+  context "when I have entered benefits" do
+    before do
+      visit estimate_build_estimate_path(estimate_id, :benefits)
+      find(:css, "#benefits-form-add-benefit-true-field").click
```

Comment:
there's a helper method for this - 
```suggestion
      select_boolean_value("benefits-form", "add-benefit", true)
```

Reply:
I hadn't spotted that. Thanks!